### PR TITLE
[#487] [TECH] Updating script to update automatically the application version (US-638).

### PR DESCRIPTION
**Pull request by jbuget**

### CODE REVIEW

File: scripts/release/prepare.sh [hunk 1]
```diff
@@ -42,6 +33,44 @@ then
...
+# https://gist.github.com/DarrenN/8c6a5b969481725a4413
+PACKAGE_VERSION=$(cat package.json \
+  | grep version \
+  | head -1 \
+  | awk -F: '{ print $2 }' \
+  | sed 's/[",]//g' \
+  | tr -d '[[:space:]]')
+
+git add . -u
+git ci -m "[RELEASE] A ${NEW_VERSION_TYPE} is being release from ${OLD_PACKAGE_VERSION} to ${PACKAGE_VERSION}."
```
Comment (jbuget) [inline scripts/release/prepare.sh]: fix "release" by "release**d**"

File: scripts/release/prepare.sh [hunk 2]
```diff
@@ -42,6 +33,44 @@ then
...
+
+git push origin dev
+
 # Fetches all last changes
 git pull
```
Comment (jbuget) [inline scripts/release/prepare.sh]: add a tag with the new version

**Comment by jbuget** [comment]
I've deployed this PR to http://638-upgrading-application-version-on-release-prepare.pix.beta.gouv.fr. Please check it out

**Comment by Akhilian** [comment]
@jbuget J'ai fait les modifications, je te laisse regarder.

**Comment by jbuget** [comment]
I've deployed this PR to http://638-upgrading-application-version-on-release-prepare.pix.beta.gouv.fr. Please check it out

**Comment by Akhilian** [comment]
[Refacto des scripts prepare.sh pour la montée de version automatique](https://trello.com/c/TiCRGZR4/638-refacto-des-scripts-preparesh-pour-la-mont%C3%A9e-de-version-automatique)

**Pull request merged**